package.json

```diff
@@ -37,8 +37,8 @@
     "lodash.isnil": "4.0.0",
     "react": "18.1.0",
     "react-dom": "18.2.0",
-    "react-native": "0.70.5",
-    "react-native-ama": "^0.6.17",
+    "react-native": "0.71.1",
+    "react-native-ama": "0.6.17",
```

Comment:
Could you update to react-native-ama 0.6.20, to avoid conflict with the ListItem PR?

Reply:
👌